lti_consumer/plugin/urls.py

```diff
@@ -4,22 +4,13 @@
 
 
 from django.conf import settings
-from django.conf.urls import url, include
-
+from django.urls import include, re_path
 from rest_framework import routers
 
-from lti_consumer.plugin.views import (
-    public_keyset_endpoint,
-    launch_gate_endpoint,
-    access_token_endpoint,
-    # LTI Advantage URLs
-    LtiAgsLineItemViewset,
-    deep_linking_response_endpoint,
-    deep_linking_content_endpoint,
-    # LTI NRPS URLs
-    LtiNrpsContextMembershipViewSet,
-)
-
+from lti_consumer.plugin.views import LtiAgsLineItemViewset  # LTI Advantage URLs; LTI NRPS URLs
```

Comment:
What are these comments for? If we are not keeping the original order, then these should be removed.

Reply:
I have made some changes to this again but I am in favor of keeping these comments as they make the urls slightly more readable and after reordering they are still there where they were intended to be.